src/typedefs.rs

```diff
@@ -23,6 +23,10 @@
 ///
 /// Trait implementation macros
 
+//TODO: Should the tag of a mac implement zeroize?
+// automatic zeroize on drop?
+// Should they implement Copy, Clone?
+
```

Comment:
The tag isn't a secret like encryption keys are, so AFAICT there's no need to zeroize on drop. But `Copy`/`Clone` may be useful.

Reply:
Fixed in https://github.com/brycx/orion/pull/99/commits/c962e9b5124307bf108a4dfb3b6a858128ca24f3